apps/astarte_housekeeping/lib/astarte_housekeeping_web/plug/metrics_plug.ex

```diff
@@ -16,8 +16,19 @@
 # limitations under the License.
 #
 
-defmodule Astarte.RealmManagementWeb.Metrics do
-  def setup do
-    Astarte.RealmManagementWeb.Metrics.PrometheusExporter.setup()
+defmodule Astarte.HousekeepingWeb.MetricsPlug do
+  import Plug.Conn
+
+  def init(_args), do: nil
+
+  def call(%{request_path: "/metrics", method: "GET"} = conn, _opts) do
+    metrics = TelemetryMetricsPrometheus.Core.scrape()
+
+    conn
+    |> put_resp_content_type("text/plain")
```

Comment:
Maybe `text/html; charset=utf-8` would be better, we should check it.

Reply:
`utf-8` is already the default charset, while the content type `text/plain` is consistent with the previous implementation.